src/platforms/python/guides/fastapi/index.mdx

```diff
@@ -80,7 +68,11 @@ Visiting `"/sentry-debug"` will trigger an error that will be captured by Sentry
 
 ## Options
 
-You can pass the following keyword arguments to `FastApiIntegration()`:
+If you want to change the default behavior of the FastAPI integration, you need to instantiate the integration by hand and pass it to Sentry's `init` function.
+
+Because FastAPI is based on the Starlette framework both integrations `StarletteIntegration`, and `FastApiIntegration` must be instantiated.
```

Comment:
```suggestion
Because FastAPI is based on the Starlette framework, both integrations `StarletteIntegration` and `FastApiIntegration`, must be instantiated.
```